tests/tools/stepper.c

```diff
@@ -113,6 +122,17 @@ int main( int argc, char **argv)
 
       if(line[0] == '\n' || line[0] == '#')
          continue;
+
+      if(strcmp(line, "bounce_connection\n") == 0) {
+         client_t *client;
+
+         LISTC_FOR_EACH(&clients, client, lnk) {
+            disconnect_cdb2h(client->db);
+         }
+
```

Comment:
`stepper_client.c` encapsulates client logic. I think it would make sense to wrap this code in a function that is defined there and then also move the definitions of `client` and `clients` back to where they were before (in general, declarations go in .h files and definitions go in .c files).

```suggestion
         clnt_disconnect_all()
```